action.yml

```
@@ -13,6 +13,9 @@ inputs:
   image:
     description: 'The URI of the container image to insert into the ECS task definition'
     required: true
+  environment-variables:
+    description: 'Variables to add to the container. Each one be on its own line and be of form NAME=value'
```

```suggestion
    description: 'Variables to add to the container. Each variable is of the form KEY=value, you can specify multiple variables with multi-line YAML strings.'
```

I think we can also add a small example to the README too to make it clear, for example in the second long example:
```
 - name: Render Amazon ECS task definition for first container
      id: render-web-container
      uses: aws-actions/amazon-ecs-render-task-definition@v1
      with:
        task-definition: task-definition.json
        container-name: web
        image: amazon/amazon-ecs-sample-1:latest
        environment-variables: |
          LOG_LEVEL=info
          ENVIRONMENT=prod
```